bundle/regal/config/config_test.rego

```diff
@@ -161,7 +161,7 @@ test_all_rules_are_in_provided_configuration if {
 		some category, title
 		data.regal.rules[category][title]
 		not endswith(title, "_test")
-		not data.regal.config.provided.rules[category][title]
+		not config.provided.rules[category][title]
```

Comment:
Incredibly satisfying when new rules find issues or improvement opportunities in Regal 😍